### PR TITLE
How to set up Svelte with vite, TypeScript and Tailwind CSS

### LINKED ISSUE #5047: How to set up Svelte with vite, TypeScript and Tailwind CSS
#### Proposed title of the article

How to set up SvelteKit with Vite, TypeScript, and Tailwind CSS 

#### Proposed article introduction

Svelte is a JavaScript front-end framework for building reactive and interactive UI components like React, Angular, and Vue. Unlike its peers would ship a JavaScript runtime to the browser to make your code work, Svelte is a compiler. It converts the declarative code that you write into imperative code that works with native browser APIs. This produces highly performant applications in a very small codebase, allowing you to write complex applications. Svelte also runs with no virtual DOM. This guide will build a portfolio application that shows how to use Vite, TypeScript, and Tailwind CSS in a Svelte application.

#### Key takeaways 

- What is Svelte
- What is Vite
- What is TypeScript
- What is Tailwind CSS
- Bootstrap a Svelte template
- Set up Vite
- Set up TypeScript
- Set up Tailwind CSS
- Build a Svelte portfolio web app using TypeScript and Tailwind CSS

#### Article quality

This guide will explain how to set up Vite, TypeScript, and Tailwind CSS in a Svelte application. We will build a simple app using Svelte with Vite, TypeScript, and Tailwind CSS.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Joakim-gakure**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5047

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Good afternoon and thank you for submitting your article for publishing. 
With the recent increase in article volume out team has been working diligently to review every article. Your article has been entered into our queue and should be reviewed (and published pending any changes) as soon as a peer reviewer is finished with the ones in the queue before it. 🚀
Thank you.

**Comment by Jethro-magaji** [comment]
@ahmadmardeni1 I would like to review this one.

**Comment by ahmadmardeni1** [comment]
Go ahead @Jethro-magaji

**Comment by Kayere** [comment]
Hello @Joakim-gakure,

 Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.

Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.

Be sure to see our resources page to see more info on plagiarism and what is considered as such.

[How to set up Svelte with vite, TypeScript and Tailwind CSS #5396 Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/7692737/How.to.set.up.Svelte.with.vite.TypeScript.and.Tailwind.CSS.5396.Copyleaks.Report.pdf)

File: content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md [hunk 1]
```diff
@@ -508,38 +305,25 @@ Call the components inside the `main` section of the app.
...
-![our-team-section](/engineering-education/svelte-with-vite-typescript-and-tailwind-css/our-team-section.PNG)
-
-![footer-section](/engineering-education/svelte-with-vite-typescript-and-tailwind-css/footer-section.PNG)
+![new-landing-page](/engineering-education/svelte-with-vite-typescript-and-tailwind-css/new-landing-page.png)
 
 ### Conclusion
-In this article, we have created a Svelte application using TypeScript and TailwindCSS. To gain more insights into the tools used, refer to the following resources.
+We used TypeScript with TailwindCSS to construct a Svelte application in this post. Refer to the following links for further information about the tools and resources used.
 
 - [Vite guide](https://vitejs.dev/guide/#overview)
```
Comment (Jethro-magaji) [inline content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md:328]: These links should be under the heading named "Additional Resources"

**Review by Jethro-magaji — COMMENTED** [review]
Hello @Joakim-gakure 
Well done for coming up with such a wonderful article. However, there're so many grammatical errors, I highly suggest using tools like Grammarly and ProWriterAid to edit your article.
Add comments inline or below your code for a better explanation.
Use 2nd person pronouns "You" instead of "We".

**Comment by Joakim-gakure** [comment]
@Jethro-magaji, I have submitted the changes above

**Comment by Joakim-gakure** [comment]
> Hello @Joakim-gakure,
> 
>  Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.
> 
> Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.
> 
> Be sure to see our resources page to see more info on plagiarism and what is considered as such.
> 
> [How to set up Svelte with vite, TypeScript and Tailwind CSS #5396 Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/7692737/How.to.set.up.Svelte.with.vite.TypeScript.and.Tailwind.CSS.5396.Copyleaks.Report.pdf)

@Kayere, I have updated the plagiarism flags, the few remaining % are HTML tags, which I can't change, they are basic boilerplate HTML tags.

File: content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-Svelte is a JavaScript front-end framework for building reactive and interactive UI components like React, Angular, and Vue. Unlike its peers would ship a JavaScript runtime to the browser to make your code work, Svelte is a compiler. It converts the declarative code that you write into imperative code that works with native browser APIs. This produces highly performant applications in a very small codebase, allowing you to write complex applications. Svelte also runs with no virtual DOM. This guide will build a portfolio application that shows how to use Vite, TypeScript, and Tailwind CSS in a Svelte application.
+Svelte is a JavaScript front-end framework for building reactive and interactive UI components like React, Angular, and Vue. Svelte is a compiler, unlike its peers, which provides a JavaScript runtime to the browser to make your code work. It converts the declarative code that you write into imperative code that works with native browser APIs. This produces highly performant applications in a very small codebase, allowing you to write complex applications.
```
Comment (Jethro-magaji) [inline content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md:1]: Rephrase to this for clarity "Svelte is a JavaScript front-end framework for building reactive and interactive UI components like React, Angular, and Vue, it is also a compiler. Unlike other frameworks, which provide a JavaScript runtime to the browser to make your code work, Svelte converts the declarative code that you write into imperative code that works with native browser APIs. This produces high-performance applications with a small codebase, allowing you to write complex applications."

File: content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
+
+This tutorial will build a portfolio application that demonstrates how to utilize Vite, TypeScript, and Tailwind CSS in a Svelte application.
```
Comment (Jethro-magaji) [inline content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md:3]: Rephrase to this for clarity "In this tutorial will build a portfolio application that shows how to use Vite, TypeScript, and Tailwind CSS in a Svelte application."

File: content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md [hunk 4]
```diff
@@ -16,8 +18,19 @@ Svelte is a JavaScript front-end framework for building reactive and interactive
   - [Set up the Team component](#set-up-the-team-component)
   - [Set up the Footer component](#set-up-the-footer-component)
 - [Conclusion](#conclusion)
+- [Additional Resources](#additional-resources)
 
 ### Setting up the application with Vite and TypeScript
+Vite is a JavaScript development server tooling that makes front-end development more easier. Vite distributes your app source files through native ES modules, allowing you to quickly start or reload your development server. Vite starts the server right away, and it takes the dependencies that don't change often and pre-bundles them using ES build.
```
Comment (Jethro-magaji) [inline content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md:24]: Rephrase to this - Vite is a JavaScript development server tool that makes front-end development easier.

File: content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md [hunk 5]
```diff
@@ -323,7 +336,15 @@ Run your development server using `npm rum dev` and open `http://localhost:3000`
 ![new-landing-page](/engineering-education/svelte-with-vite-typescript-and-tailwind-css/new-landing-page.png)
 
 ### Conclusion
-We used TypeScript with TailwindCSS to construct a Svelte application in this post. Refer to the following links for further information about the tools and resources used.
+This guide has illustrated how you set up a TypeScript Svelte boilerplate that you can use to write TypeScript code within a Svelte application. We also configured and used TailwindCSS to create a Svelte portfolio application.
```
Comment (Jethro-magaji) [inline content/articles/svelte-with-vite-typescript-and-tailwind-css/index.md:420]: Please, include links to the GitHub repo and live demo site for a better understanding.

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi @Joakim-gakure 
I have dropped some reviews for you. Please use Grammarly and ProWritingAid to edit your article as there are still some grammatical errors. Else the next revision will require the closure of this PR.
Thanks.

**Comment by WanjaMIKE** [comment]
Plagiarism check done.

**Comment by ahmadmardeni1** [comment]
Can you give me a status update? @Jethro-magaji Is this article ready for a final review?

**Comment by Jethro-magaji** [comment]
Hi! @ahmadmardeni1
The Author has made the required changes, and I have done my part of the review, please complete the review process by adding front matter and other necessary tags.
Cc @WanjaMIKE
Thanks.

**Comment by WanjaMIKE** [comment]
Nice work gents @Joakim-gakure @Jethro-magaji

**Comment by WanjaMIKE** [comment]
Merged #5396 into master

**Comment by Jethro-magaji** [comment]
I have just finished my final test review.
cc @WanjaMIKE @section-engineering-education/content-moderator-team @ahmadmardeni1 

Screenshot (868)

**Pull request merged**